.gitignore

```
@@ -51,6 +51,7 @@ components/version/version.ts
 components/version/version.tsx
 components/version/token.json
 components/version/token-meta.json
+components/version/depreciated-version.json
```

```suggestion
components/version/deprecated.json
```

那改成 deprecated.json 就行，前面有 version 了。